packages/shaders/src/shader-mount.ts

```diff
@@ -165,76 +178,74 @@ export class ShaderMount {
     });
 
     this.resizeObserver.observe(this.parentElement);
-    visualViewport?.addEventListener('resize', this.handleVisualViewportChange);
-
-    const rect = this.parentElement.getBoundingClientRect();
-    this.parentWidth = rect.width;
-    this.parentHeight = rect.height;
-    this.handleResize();
```

Comment:
We didn't really need to do this, the resize observer callback runs after instantiation anyway (and now that we use real device pixels, we'd need to wait for it anyway)